nextjs-13/src/repositories/newsletter.repository.test.ts

```diff
@@ -0,0 +1,24 @@
+import { dbClientMock } from '@test/database';
+import { newsletterFactory } from '@test/factories/newsletter.factory';
+
+import { createNewsletter } from './newsletter.repository';
+
+describe('Newsletter Respository', () => {
+  describe('createNewsletter', () => {
+    it('creates a newsletter', async () => {
+      const newsletterAttributes = {
+        name: 'iOS 17 release',
+        content:
+          'iOS 17 will reportedly have a new feature that turns locked iPhones into smart home displays. The interface will display information such as appointments, the weather, and notifications. It will operate similarly to smart home devices from other companies.',
+        user: { connect: { id: '1' } },
```

Comment:
Is `connect` required by Prisma? I am unsure how this works for association 🤔

Reply:
Yes it is, here are [some examples](https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#examples-25).